platform-sdk/swirlds-common/src/main/java/com/swirlds/common/system/address/AddressBookUtils.java

```diff
@@ -30,12 +30,20 @@
 /**
  * A utility class for AddressBook functionality.
  * <p>
- * Format for the config.txt address book: keywork "address" followed by a comma, followed by the node ID, followed by a
- * comma, followed by the nickname, followed by a comma, followed by the self name, followed by a comma, followed by the
- * weight, followed by a comma, followed by the internal IP address, followed by a comma, followed by the internal port,
- * followed by a comma, followed by the external IP address, followed by a comma, followed by the external port,
- * followed by a comma, followed by the memo.  The memo is optional.  Each address line is terminated by a newline ("\n"
- * or "\r\n") or end of file.
+ * Each line in the config.txt address book contains the following comma separated elements: config.txt address book: keywork "address" followed by a comma, followed by the node ID, followed by a
```

Comment:
It looks like there is a fragment of the old javadoc hanging around at the end of this line.
```suggestion
 * Each line in the config.txt address book contains the following comma separated elements:
```